plugins/arangodbendorsements/main.go

```diff
@@ -594,23 +611,17 @@ func (e *ArangoEndorsementStore) GetAltSoftwareComponents(args common.QueryArgs)
 	return result, nil
 }
 
-// InputSwidMatches checks whether a given SWID is present in a SWID list
```

Comment:
Yes, please remove this function, as this is left here due to code refactorization.